curriculum/challenges/english/blocks/zh-a1-warm-up-meeting-new-teammates/68fe153a6494282327edcb3a.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 68fe153a6494282327edcb3a
+title: Task 4
+challengeType: 22
+dashedName: task-4
+---
+
+<!-- (Audio) Wang Hua: 请问 (qǐng wèn) -->
+
+# --description--
+
+Wang Hua wants to politely ask someone's name. She uses a word before asking a question to show respect.
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`BLANK`
```

Comment:
Should this be one or two blanks?

And do you think we should always have one blank for each character, or that would depend on the context?